src/main/java/nl/ordina/jobcrawler/service/VacancyStarter.java

```diff
@@ -73,4 +69,25 @@ public void scrape() throws IOException {
         log.info("Finished scraping");
         allVacancies.clear();
     }
+
+    @Scheduled(cron = "0 30 11,17 * * *") // Runs two times a day. At 11.30am and 5.30pm.
+    public void deleteNonExistingVacancies() {
+        log.info("CRON Scheduled -- Started deleting non-existing jobs");
+        List<Vacancy> allVacancies = vacancyService.getAllVacancies();
+        List<Vacancy> vacanciesToDelete = allVacancies.stream()
+                .filter(vacancy -> {
+                    if (vacancy.getBroker().equals("Yacht"))
+                        return !yachtVacancyScraper.doesVacancyExist(vacancy.getVacancyURL());
```

Comment:
Although not strictly necessary, using curly braces will make the code more robust. Not everyone will notice the lack of curly braces, and some developers will assume that they are in a code block, which would result in unexpected behavior.

Reply:
done